eclipse/ca.mcgill.sable.soot/src/ca/mcgill/sable/soot/ui/PhaseOptionsDialog.java

```diff
@@ -9119,18 +9122,6 @@ private Composite Output_OptionsCreate(Composite parent) {
 						"1.9 9",
 						"\nForce Java 1.9 as output version. (Experimental)",
 						false),
-				new OptionData("Java 1.10",
```

Comment:
Why are these removed? Werent they added by the default branch?

Reply:
MIght it be that they get re-added when you call ``mvn clean verify`?

Reply:
Ran mvn clean verify. But did not get any changes. Will check this

Reply:
I see this has been removed in the develop branch https://github.com/soot-oss/soot/blob/e233653f2a1bd823d20b490ff75ccc50033ff421/eclipse/ca.mcgill.sable.soot/src/ca/mcgill/sable/soot/ui/PhaseOptionsDialog.java#L9122

Reply:
No! This was added in the default branch! Thats why it is not in develop.

Reply:
Okay. Will add them.